features/index.md

```diff
@@ -0,0 +1,279 @@
+---
```

Comment:
I'm not sure a `features/` folder at the root makes sense?

Reply:
I am actually not sure why Marina created this folder. This features folder doesn't seem to be used for anything, as the links are outdated and we have features folders in the checkout, payment menu and each payment instrument sections.

Reply:
Then delete it?